runc.go

```diff
@@ -593,6 +594,10 @@ func parseVersion(data []byte) (Version, error) {
 }
 
 func (r *Runc) args() (out []string) {
+	if len(r.RuntimeArgs) != 0 {
+		return r.RuntimeArgs
```

Comment:
I don't like this because its bypasses every other argument.  I think there should be a better way to handle this, especially in your other examples you are wanting to pass the `--debug` flag which we already have on the Runc struct.